From 23d5561562093779424de55cffc3fba15e34ca6f Mon Sep 17 00:00:00 2001
From: Sun peng Li <Sunpeng.Li@amd.com>
Date: Tue, 2 May 2017 17:09:37 -0400
Subject: [PATCH 0405/4131] drm/amd/display: Reverting locks around updating
 freesync property

Revert "SWDEV-114487 - dm/amdgpu: Add lock around updating freesync property"

This reverts commit b54fd07f238a01f1155c9e2b378e148e5df2c548.

Reviewed-by: Jordan Lazare <Jordan.Lazare@amd.com>
Acked-by: Harry Wentland <Harry.Wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
index 9c794cb..a5218b6 100644
--- a/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
+++ b/drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_types.c
@@ -1214,7 +1214,6 @@ static int amdgpu_freesync_update_property_atomic(
 	dev  = connector->dev;
 	adev = dev->dev_private;
 
-	drm_modeset_lock(&dev->mode_config.connection_mutex, NULL);
 	ret = drm_object_property_get_value(
 			&connector->base,
 			adev->mode_info.freesync_property,
@@ -1224,8 +1223,6 @@ static int amdgpu_freesync_update_property_atomic(
 				&connector->base,
 				adev->mode_info.freesync_property,
 				val_capable);
-	drm_modeset_unlock(&dev->mode_config.connection_mutex);
-
 	return ret;
 
 }
-- 
2.7.4

